CHANGES.rst

```diff
@@ -2,6 +2,23 @@
 Changelog
 ---------
 
+1.2.1
+*****
+- Bug-fixes for service version 1.0, after pilot integration of new service at BL
+
+1.2.0
+*****
+- first implementation of federal extract requirements as per november 2017
+  (service in version 1.0, extract in version 1.0.1, data model in version 1.0.1)
+- static extract implementation update according to Weisung july 1st, 2018
+- oereb lex model creation support and documentation
```

Comment:
oereb lex **version 1.1.0** model creation... ?
Or it is not useful ?

Reply:
in my eyes, there is no real versioning of oereb lex - any custom models (including models with oereblex usage) need to conform to the model scheme used in that version of pyramid_oereb.